config/router.js

```diff
@@ -9,7 +9,24 @@ module.exports = (router) => {
 
     router.del = router.delete;
 
-    router.get('/user/:id', __.user.get_user);
+    router.route('/user/:id')
```

Comment:
Be consistent in using whitespaces (tabs and spaces). Hindi ata pantay-pantay ung mga spaces.